### PR TITLE
chore: remove story wrapper and add prop for demos

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

**Comment by aromko** [comment]
Is there some context what and why we are doing this?

**Comment by sarahgm** [comment]
> Is there some context what and why we are doing these changes?

Yes you can go to our docs and storybook, then you will see that the Table Checkboxes are aligend with form fields. There are no because its a table but around the demo there is the `<FieldGroup>` which sets the labelwidth to 100px, and because of that the Checkbox also sets the labelWidth.

For Storybook i tried to solve it through parameteres and addons but they are not allowing custom stuff there, so I removed the wrapper and add the `<FieldGroup>` to all Form Fields manuel

**Comment by aromko** [comment]
> > Is there some context what and why we are doing these changes?
> 
> @aromko Yes you can go to our docs and storybook, then you will see that the Table Checkboxes are aligend with form fields. There are no because its a table but around the demo there is the `<FieldGroup>` which sets the labelwidth to 100px, and because of that the Checkbox also sets the labelWidth.
> 
> For Storybook i tried to solve it through parameteres and addons but they are not allowing custom stuff there, so I removed the wrapper and add the `<FieldGroup>` to all Form Fields manuel

💡 thanks

**Comment by sebald** [comment]
@sarahgm I would take over so we can move this forward. Can you give me what needs to be done here tomorrow? :D

File: packages/components/src/Autocomplete/Autocomplete.stories.tsx [hunk 1]
```diff
@@ -98,23 +101,25 @@ export const Controlled: Story = {
...
-              Harry Potter
-            </Autocomplete.Item>
-            <Autocomplete.Item id="lord-of-the-rings">
-              Lord of the Rings
-            </Autocomplete.Item>
-            <Autocomplete.Item id="star-wars">Star Wars</Autocomplete.Item>
-            <Autocomplete.Item id="star-trek">Star Trek</Autocomplete.Item>
-            <Autocomplete.Item id="firefly">Firefly</Autocomplete.Item>
-          </Autocomplete>
+          <FieldGroup labelWidth="200px">
```
[the remark this answers is not in the document]
Comment (sarahgm) [reply packages/components/src/Autocomplete/Autocomplete.stories.tsx:104]: Hm I'm not sure I understand correctly. 
If I get the title than what should happen?  (If title in ???)

[the remark this answers is not in the document]
Comment (sarahgm) [reply packages/components/src/Autocomplete/Autocomplete.stories.tsx:104]: @sebald @OsamaAbdellateef can you maybe explain what he meant? :(

[the remark this answers is not in the document]
Comment (sarahgm) [reply packages/components/src/Autocomplete/Autocomplete.stories.tsx]: I added a parameter called fieldGroup and check if its set if so than add a fieldgroup. I hope this is what you meant :D

**Review by sebald — COMMENTED** [review]
Honest question 🙈 We now have a "positive implementation" (`fieldGroup: true`) and a "negative one" (`disableLabelWidth: true`). I think we should settle on one, otherwhise this will get very confusing for ust.

**Comment by sarahgm** [comment]
> Honest question 🙈 We now have a "positive implementation" (`fieldGroup: true`) and a "negative one" (`disableLabelWidth: true`). I think we should settle on one, otherwhise this will get very confusing for ust.

There would be the thing than that you have to set to every demo `fieldGroup: true` to every demo that has a fieldgroup. For me I thought it easier to just disable the fieldGroup for the table. 
Same for the storybook preview, there I found it easier to just say these components should have a fieldGroup.

And to not forget, I would add this information to out Component Developing Guidelines :) 

@OsamaAbdellateef  what do you think about this?

**Comment by sebald** [comment]
> > Honest question 🙈 We now have a "positive implementation" (`fieldGroup: true`) and a "negative one" (`disableLabelWidth: true`). I think we should settle on one, otherwhise this will get very confusing for ust.
> 
> There would be the thing than that you have to set to every demo `fieldGroup: true` to every demo that has a fieldgroup. For me I thought it easier to just disable the fieldGroup for the table. Same for the storybook preview, there I found it easier to just say these components should have a `fieldGroup:true`.
> 
> And to not forget, I would add this information to our Component Developing Guidelines :)
> 
> And if you still don't like it, maybe I can use the same `disableLabelWidth: true` just in the table story?
> 
> @OsamaAbdellateef @sebald what do you think about this?

Yes, disabling it would also be fine. I just don't like to have different options for the same thing :D

**Review by sebald — APPROVED** [review]

**Pull request merged**